staff/daniel-villegas/pre-course/all-projects/project-1/scripts.js

```diff
@@ -0,0 +1,37 @@
+function calculator(n1,n2){
+ 
+    var sum = n1+n2;
+    var minus = n1-n2;
+    var plus = n1*n2;
+    var division = n1/n2;
+ 
+    for (var o = 0 ; o == 4; o++);
+ 
+    //Si los valores no son numeros, ergo true.
+    if (isNaN(n1) == true && isNaN(n2) == true){
```

Comment:
I would rather use strict comparison by using "===" instead of "==" to avoid implicit value transformation when comparing values.

The calculator is amazing though. Kudos!!!

https://es.stackoverflow.com/questions/331/diferencias-entre-y-comparaciones-en-javascript

Reply:
Thank u for the advice, I'll change right now! 👍